pkg/client/cli/cmd_status.go

```diff
@@ -240,9 +248,12 @@ func (cs *userDaemonStatus) WriteTo(out io.Writer) (int64, error) {
 		n += ioutil.Printf(out, "  Version           : %s (api %d)\n", cs.Version, cs.APIVersion)
 		n += ioutil.Printf(out, "  Executable        : %s\n", cs.Executable)
 		n += ioutil.Printf(out, "  Install ID        : %s\n", cs.InstallID)
-		n += ioutil.Printf(out, "  Status            : %s\n", cs.Status)
-		n += ioutil.Printf(out, "    Mode            : %s\n", cs.Mode)
-		n += ioutil.Printf(out, "    Client Count    : %d\n", cs.ClientCount)
+		if tms := cs.TrafficManagerStatus; tms != nil {
+			n += ioutil.Println(out, "Traffic Manager: Running")
+			n += ioutil.Printf(out, "  Status            : %s\n", cs.Status)
+			n += ioutil.Printf(out, "    Mode            : %s\n", tms.Mode)
+			n += ioutil.Printf(out, "    Client Count    : %d\n", tms.Mode)
+		}
```

Comment:
It seems a bit overkill to both report that the traffic-manager is running and that it is connected. Connected implies running. How about this?
```suggestion
		if tms := cs.TrafficManagerStatus; tms != nil {
			n += ioutil.Println(out, "Traffic Manager: Connected")
			n += ioutil.Printf(out, "  Mode            : %s\n", tms.Mode)
			n += ioutil.Printf(out, "  Client Count    : %d\n", tms.Mode)
		} else {
			n += ioutil.Println(out, "Traffic Manager: Not connected")
		}
```

Reply:
yeah that looks better :+1: